service_catalog/api/views/approval_workflow_api_views.py

```diff
@@ -78,14 +79,16 @@ def reject(self, request, pk=None):
         target_request = get_object_or_404(Request, id=pk)
         if target_request.approval_workflow_state.current_step is None:
             return Response({"error": "No pending step to approve"}, status=HTTP_404_NOT_FOUND)
-        if not request.user.has_perm(target_request.approval_workflow_state.current_step.approval_step.permission,
+        if not request.user.has_perm(target_request.approval_workflow_state.current_step.approval_step.permission.get_permission_str(),
                                      target_request):
             raise PermissionDenied
-        target_request.approval_workflow_state.reject_current_step(request.user)
+        # reject the current step if exist
+        if target_request.approval_workflow_state is not None:
+            target_request.approval_workflow_state.reject_current_step(request.user)
         # reject the request
         target_request.reject(request.user)
         target_request.save()
-        # send_mail_request_update(target_request, user_applied_state=request.user, message=message)
```

Comment:
message are no more displayed in notifications?

Reply:
It's an error, it should be